### ISSUE TITLE
Pomegranate error when clicking Run with REPL

### ISSUE THREAD

**Issue by adamhoward** [oakes/Nightcode#234]
When I click Run with REPL on a brand new Console Project, the REPL starts, spits out the following stacktrace, then never shows a prompt.

```
Exception in thread "main" java.lang.IllegalStateException: Could not find a suitable classloader to modify from clojure.lang.LazySeq@d7415eeb
	at cemerick.pomegranate$add_classpath.invokeStatic(pomegranate.clj:54)
	at cemerick.pomegranate$add_classpath.invoke(pomegranate.clj:42)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:659)
	at clojure.core$apply.invoke(core.clj:652)
	at cemerick.pomegranate$add_dependencies.invokeStatic(pomegranate.clj:85)
	at cemerick.pomegranate$add_dependencies.doInvoke(pomegranate.clj:57)
	at clojure.lang.RestFn.invoke(RestFn.java:457)
	at leiningen.repl$client.invokeStatic(repl.clj:261)
	at leiningen.repl$client.invoke(repl.clj:258)
	at leiningen.repl$repl.invokeStatic(repl.clj:320)
	at leiningen.repl$repl.doInvoke(repl.clj:267)
	at clojure.lang.RestFn.invoke(RestFn.java:425)
	at leiningen.repl$repl.invokeStatic(repl.clj:307)
	at leiningen.repl$repl.invoke(repl.clj:267)
	at nightcode.lein$lein_BANG_.invokeStatic(lein.clj:32)
	at nightcode.lein$lein_BANG_.invoke(lein.clj:27)
	at nightcode.lein$_main.invokeStatic(lein.clj:39)
	at nightcode.lein$_main.doInvoke(lein.clj:37)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at nightcode.lein.main(Unknown Source)
```

If I run `lein repl` from the command line in the project folder, the REPL starts with no errors.

### Environment
- macOS 10.12.6
- Java 1.8.0.144
- Leiningen 2.8.1
- Nightcode 2.5.2

**Comment by joinr** [comment]
I get the same error on Windows 10, 64-bit java 8_152, nightcode 2.5.2.
Looks like some folks on the lein issues were getting similar problems, and related it to lack of plugins?  Weird.  2.8.1 of lein seems to have a couple of issues for me, this among them.

**Comment by oakes** [comment]
Thanks for pointing this out, I'll release a fix today when I get the chance.

**Comment by oakes** [comment]
I just released version 2.5.3 which fixes this issue. Thanks!

**Comment by joinr** [comment]
I just popped another one (now that I have occasion to pop open NightCode
Proper), sorry man.
